### PR TITLE
 SettingUp.md: add instructions on how to login as a student #1745

### PR DESCRIPTION
**Pull request by drunken-saint**
...sts

Added an instruction in settingUp.md & made other changes to pass the
tests

### CODE REVIEW

File: src/main/webapp/WEB-INF/classes/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+/META-INF
```
Comment (kanghj) [inline src/main/webapp/WEB-INF/classes/.gitignore]: You should remove changes that are not related to the issue.You mentioned making changes to pass the tests, but it is unlikely that you need to do that. The test failures should be caused by other reasons.

**Comment by kanghj** [comment]
There is a format to follow regarding the pull request name, which can be found [here](https://github.com/TEAMMATES/repo/blob/master/devdocs/process.md). 

In the teammates contributors google group, you said that 2 tests were still failing. Can I ask which 2 tests are still failing for you?

File: devdocs/settingUp.md [hunk 2]
```diff
@@ -75,7 +75,11 @@ Important: When a version is specified, please install that version instead of t
    Instead, you can use the join link given after adding an intructor, to complete the joining process.<br>
    Remember to change the URL of the link if necessary.<br>
    e.g. Change `http://teammates-john.appspot.com/page/instructorCourseJoin?...` to `http://localhost:8888/page/instructorCourseJoin?...`
-      
+8. Now, to access the dev server as a student , first make sure you are logged in as an instructor. Add a course for yourself and then add the students for the course .<br>
+   After that , login as admin and search for the student you added in as instructor . From the search results , use the join link adjacent to the student you added. <br>
+   then , logout and use that join link to login as a student . You have the required access now. <br>
+   (Make sure you use the `http://localhost:8888/` as the host instead of the one given in the join link) <br>   
+   Alternative : Run the test cases , the test case creates several student account in the  data store . Use one of them to login. <br>
```
Comment (kanghj) [inline devdocs/settingUp.md]: You should follow the same punctuation as the rest of document and place punctuation marks immediately after words. 
e.g. "...as a student, first make sure..." as opposed to "...as a student , first make sure...".

"datastore" should also be a single word

**Comment by drunken-saint** [comment]
I ran the tests again . And now its failing even more tests . There must be some errors I might have made during the process . I think I should set up the project one more time again from the start . Whats your opinion? 
I'll delete this branch and make a new branch after cloning a fresh repo on my pc .

**Comment by kanghj** [comment]
Yes, you can try that. Follow the instructions carefully and check the troubleshooting guide if needed.

**Comment by drunken-saint** [comment]
I made another pull request . Please check the following link :)
https://github.com/TEAMMATES/repo/pull/2928

**Pull request closed without merging**